Packs/OpenCTI/Integrations/OpenCTI/OpenCTI.yml

```diff
@@ -293,12 +300,350 @@ script:
+    description: Delete incident.
+    name: opencti-incident-delete
+  - arguments: []
+    description: Get a list of all incident types.
+    name: opencti-incident-types-list
+    outputs:
+    - contextPath: OpenCTI.IncidentTypes.IncidentTypesList.id
+      description: Incident type ID.
+    - contextPath: OpenCTI.IncidentTypes.IncidentTypesList.name
+      description: Incident type name.
+    - contextPath: OpenCTI.IncidentTypes.IncidentTypesList.description
+      description: Incident type description.
+  - arguments:
+    - description: Name of the indicator.
+      name: name
+      required: true
+      type: string
+    - description: Value of the indicator.
+      name: indicator
+      required: true
+      type: string
+    - auto: PREDEFINED
+      description: Main observable type for the indicator.
+      name: main_observable_type
+      predefined:
+      - Account
+      - Domain
+      - Email
+      - File-MD5
+      - File-SHA1
+      - File-SHA256
+      - IP
+      - IPv6
+      - Registry Key
+      - URL
+      required: true
+      type: string
+    - description: The types of the indicator. Use opencti-indicator-types-list to find all indicator types in OpenCTI.
+      isArray: true
+      name: indicator_types
+      type: string
+    - description: The description of the indicator.
+      name: description
+      type: string
+    - defaultValue: "50"
+      description: Confidence level for the indicator, value between 0 and 100. Default is 50.
+      name: confidence
+    - defaultValue: "50"
+      description: The score of the indicator, value between 0 and 100. Default is 50.
+      name: score
+    - description: The valid-from date for the indicator in the format YYYY-MM-DDThh:mm:ss.sssZ.
+      name: valid_from
+      type: string
+    - description: The valid-until date for the indicator in the format YYYY-MM-DDThh:mm:ss.sssZ.
+      name: valid_until
+      type: string
+    - description: The ID of the entity that created the indicator (use opencti-organization-list to find or create).
+      name: created_by
+      type: string
+    - description: The label ID for the indicator (use opencti-label-list to find or create).
+      name: label_id
+      type: string
+    - description: The marking ID for the indicator (use opencti-marking-definition-list to find).
+      name: marking_id
+      type: string
+    - description: External references ID for the indicator (use opencti-external-reference-create to create).
+      name: external_references_id
+      type: string
+    - auto: PREDEFINED
+      defaultValue: "false"
+      description: Create OpenCTI observable related with the OpenCTI indicator created.
+      name: create_observables
+      predefined:
+      - "true"
+      - "false"
+    description: Create a new indicator in OpenCTI.
+    name: opencti-indicator-create
+    outputs:
+    - contextPath: OpenCTI.Indicator.id
+      description: New Indicator ID.
+      type: string
```

Comment:
Regarding the following validation error RM102:
>opencti-indicator-create:
The following outputs are missing from yml: OpenCTI.Indicator.value, OpenCTI.Indicator.type

The validation's description validates that all context outputs defined in the README file are present in the YML file, and vice versa.
Where the rational is:
>"Ensuring consistency between the README and YML files helps maintain accurate documentation and prevents discrepancies."

So basically there is a consistency between the readme:
https://github.com/demisto/content/blob/bf19c98a05b0af567658e6e6624091b3cda295d6/Packs/OpenCTI/Integrations/OpenCTI/README.md?plain=1#L154
And the yml file which only have OpenCTI.Indicator.id in the context output:
https://github.com/demisto/content/pull/37270/files#diff-3a03284b240156d148b67fe01f470c1da9010d62638df25c8b68e4b5e184b4b2R428

It also relevant for `opencti-get-indicators`:
>opencti-get-indicators:
The following outputs are missing from readme: OpenCTI.Indicators.IndicatorList.creators